superduperdb/base/logger.py

```diff
@@ -24,14 +24,24 @@ class Logging:
         logger.configure(handlers=[{"sink": custom_handler, "serialize": True}])
     else:
         logger.remove()
-        fmt = "<green>{time:YYYY-MMM-DD HH:mm:ss}</green> | <level>{level: <8}</level> | <cyan>{name}</cyan>:<cyan>{line}</cyan> | <cyan>{extra}</cyan> <level> | {message} </level> "
+        fmt = (
+            "<green>{time:YYYY-MMM-DD HH:mm:ss}</green> | <level>{level: <8}</level> |"
+            "<cyan>{name}</cyan>:<cyan>{line}</cyan> |"
+            " <cyan>{extra}</cyan> <level> | {message} </level>"
+        )
 
         # Send "ERROR" and below to stdout, with "ERROR" and above copied to stderr
         logger.add(stdout, format=fmt, level=CFG.logging.level)
         logger.add(stderr, format=fmt, level=LogLevel.ERROR)
 
         # Send "ERROR" and below to stdout, with "ERROR" and above moved to stderr
-        # logger.add(sys.stdout, filter=lambda record: record["level"].no < 40, level="INFO")
+        # import sys
```

Comment:
do we need executable comments?

Reply:
This is just for "convenience" until we make a decision about the logging output.
(if an error happens, do we want it to be printed on stdout, stderr, or on both ?)

Reply:
I see,
well its fine with me this way

but see if we can make it like

```python

LOG_ERROR = False
if LOG_ERROR:
    logger.add(
            sys.stdout,
            filter=lambda record: record["level"].no < 40,
            level="INFO",
         )
```